### ISSUE TITLE
Recommended vs Configured Feature Values

### ISSUE THREAD

**Issue by dataCollegechurch** [MicrosoftDocs/microsoft-365-docs#7272]
In comment column for this value: "Redirect attachment with detected attachments : Enable redirect". It indicates this "Note: This setting is not configured in the Standard, Strict, or Built-in protection preset security policies. The Standard and Strict values indicate our recommended values"

As noted in previous feedback this applies to the value "Enable users to protect (impersonated user protection)" within the Impersonation settings section and possibly others as well. You are not able to update the list of protected users within the Standard or Strict policy, so a setting value of "true" is misleading/confusing here.

It also appears that while the Standard and Strict value for AutoForwardingMode is set to "Automatic - System-controlled", this setting can be overridden by a custom policy that turns autoforwarding on. Typically custom polices cannot override Standard and Strict polices.

Can the document be updated to indicated which values are recommended and which values are actually configured by the Standard and Strict policies?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0c77e1cb-1509-83cc-7c2a-564d5e3e91ea
* Version Independent ID: 9f07278c-37ab-beac-a5dd-fa4f4825b3a4
* Content: [Microsoft recommendations for EOP and Defender for Office 365 security settings - Office 365](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365?view=o365-worldwide#eop-outbound-spam-policy-settings)
* Content Source: [microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365.md)
* Product: **m365-security**
* Technology: **mdo**
* GitHub Login: @MSFTTracyP
* Microsoft Alias: **tracyp**

**Comment by yogkumgit** [comment]
@MSFTTracyP Please help us in resolving this issue. Thanks

**Comment by chrisda** [comment]
@dataCollegechurch, I looked into this closely, and I made some updates:

- Outbound spam policies are not affected by preset security policies. No specific policy is created for Standard or Strict, so only the default outbound spam policy and any custom outbound spam policies are in play. I added the following note to the section:

  > [!NOTE]
  > Outbound spam policies are not part of Standard or Strict preset security policies. The **Standard** and **Strict** values indicate our **recommended** values in the default outbound spam policy or custom policies that you create.

- In impersonation settings for MDO anti-phishing policies, I added clarifications for some of the settings as you indicated. For example, impersonated user protection is turned on by default in Standard and Strict preset security policies, but obviously no users can be specified, so the **If message is detected as an impersonated user** action effectively does nothing, even though the default value is Quarantine for both Standard and Strict preset security policies. Similar situation with impersonated domain protection, but **Include domains I own** is enabled by default in Standard and Strict preset security policies. So the **If message is detected as an impersonated domain** only affects your accepted domains in Standard and Strict preset security policies.

That was all I could find that needed clarification.
